snap/snapcraft.yaml

```diff
@@ -12,7 +12,7 @@ apps:
   universal-ctags:
     command: ctags
     plugs:
-      - home
+      - personal-files
```

Comment:
You need to add a top-level `plugs` block that details which personal files you want to be exposed by your usage of the `personal-files` interface. You also need to call it something other than `personal-files`. [See the documentation](https://docs.snapcraft.io/personal-files-interface) for more information.